_includes/v23.1/known-limitations/sql-cursors.md

```diff
@@ -0,0 +1,25 @@
+CockroachDB implements SQL {% if page.name == "known-limitations.md" %} [cursor](cursors.html) {% else %} cursor {% endif %} support with the following limitations:
+
+- `DECLARE` only supports forward cursors. Reverse cursors created with `DECLARE SCROLL` are not supported. [cockroachdb/cockroach#77102](https://github.com/cockroachdb/cockroach/issues/77102)
```

Comment:
These Vale failures, man. Annoying. If you want to have Vale ignore the link text to CockroachDB GitHub issues, add the following to `vale.ini`:

~~~
TokenIgnores = (\[cockroachdb/cockroach.*?\])
~~~

Reply:
Thanks for this info, added that line to the `.vale.ini` file

Reply:
I'm gonna investigate this locally. Vale shouldn't be this unnecessarily chatty.

Reply:
Yeah, that's a good fix.